stdlib/LibGit2/src/utils.jl

```diff
@@ -107,6 +111,11 @@ provided the URL produced will use the alternative [scp-like syntax](https://git
     provided. Cannot be specified when using the scp-like syntax.
   * `path::AbstractString=""`: the path to use in the output if provided.
 
+!!!warning
```

Comment:
This needs a space between `!` and `warning` for it to be recognized correctly. Currently it shows up literally as `!!!warning` in the docstring rather than being rendered nicely.